data/json/vehicles/custom_vehicles.json

```diff
@@ -0,0 +1,114 @@
+[
+    {
+        "type" : "vehicle",
+        "id" : "hearse",
```

Comment:
I'd prefer to move this vehicle to any (appropriate) existing files or at least rename the file.